pyfstat/core.py

```diff
@@ -1076,108 +1075,89 @@ def plot_twoF_cumulative(
         label, outdir : str
         add_pfs : bool
             If true, plot the predicted 2F and standard deviation
-        N : int
-            Number of points to use
-        injectSources : dict
-            See `ComputeFstat`
-        ax : matplotlib.axes._subplots_AxesSubplot, optional
-            Axis to add the plot to.
-        c : str
-            Colour
+        injection_parameters: dict
+            Dictionary with injection parameters to predict Fstat.
+            At least (h0, cosi, psi, Alpha, Delta, F0) keys must be
+            present (any other key will simply be ignored).
+        predict_fstat_segments : int
+            Number of points to use for PredictFStat.
         savefig : bool
             If true, save the figure in outdir
-        title, plt_label: str
-            Figure title and label
 
         Returns
         -------
-        tauS, tauF : ndarray shape (N,)
+        tauS, tauF : ndarray shape (predict_fstat_segments,)
             If savefig, the times and twoF (cumulative) values
         ax : matplotlib.axes._subplots_AxesSubplot, optional
             If savefig is False
 
         """
-        if ax is None:
-            fig, ax = plt.subplots()
-        if injectSources:
+
+        taus, twoFs = self.calculate_twoF_cumulative(**calculate_twoF_cumulative_kwargs)
+        taus_days = taus / 86400.0
+
+        axis_kwargs = {
+            "xlabel": r"Days from $t_{{\rm start}}={:.0f}$".format(
+                calculate_twoF_cumulative_kwargs["tstart"]
```

Comment:
If they're optional kwargs, for safety this should be something like
```suggestion
                calculate_twoF_cumulative_kwargs.get("tstart",0)
```
though I'm not sure if the previous `self.calculate_twoF_cumulative` won't have failed anyway if that argument is missing...?

Reply:
```
>>> np.max([1, None])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<__array_function__ internals>", line 5, in amax
  File "/home/rodrigo/gravity/venv/pyfstat/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 2705, in amax
    return _wrapreduction(a, np.maximum, 'max', axis, None, out,
  File "/home/rodrigo/gravity/venv/pyfstat/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 87, in _wrapreduction
    return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
TypeError: '>=' not supported between instances of 'int' and 'NoneType'
```
I usually don't like this type of coupling (it's lurking bug, it's going to fail whenever `calculate_twoF_cumulative` changes a bit), so we should do something sane about it (I can't conclude whether your proposal is the one that makes sense or not right now).